packages/server/src/arpa_reporter/db/uploads.js

```diff
@@ -145,6 +149,19 @@ async function markNotValidated(uploadId, trns = knex) {
         .update({
             validated_at: null,
             validated_by: null,
+            invalidated_at: null,
+            invalidated_by: null,
+        })
+        .returning('*')
+        .then((rows) => rows[0]);
+}
+
+async function markInvalidated(uploadId, userId, trns = knex) {
+    return trns('uploads')
+        .where('id', uploadId)
+        .update({
```

Comment:
we should also null the validated fields here-
```js
.update({
    validated_at: null,
    validated_by: null,
    ...
```

Reply:
I was debating that, but would it be valuable to know if it was validated then manually invalidated?